### ISSUE TITLE
JupyterLab file browser entries empty with this extension installed

### ISSUE THREAD

**Issue by 02JanDal** [jupyterlab/jupyterlab-git#411]
## Description

With `jupyterlab-git` installed all entries in the file browser are empty (but they are still there and clickable, just... empty...). It's still possible to navigate (though you can't really tell where you're going) and opening files, and when changing directory the current path is shown correctly.

![screenshot](https://user-images.githubusercontent.com/3686998/64616100-1e87e880-d3dc-11e9-82ab-2bb0323ca7b2.png)

## Reproduce

1. Install jupyterlab-git in a docker image based on base-notebook (from docker-stacks)
2. Launch via JupyterHub
3. File browser entries are empty

## Expected behavior

File browser entries with name, icon and size.

## Context

- Python package version: 0.8.1 (as installed by `pip install --upgrade jupyterhub-git
- Extension version:
```
JupyterLab v1.0.4
Known labextensions:
   app dir: /opt/conda/share/jupyter/lab
        @jupyterlab/git v0.8.2  enabled  OK
        nbdime-jupyterlab v1.0.0  enabled  OK
```

<details><summary>Command Line Output</summary>
<pre>
Set username to: jandal,
Set jandal UID to: 1002,
Executing the command: jupyterhub-singleuser --ip=0.0.0.0 --port=8888 --notebook-dir=/ --NotebookApp.default_url=/lab --debug,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:177] Searching ['/home/jandal', '/home/jandal/.jupyter', '/opt/conda/etc/jupyter', '/usr/local/etc/jupyter', '/etc/jupyter'] for config files,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:555] Looking for jupyter_config in /etc/jupyter,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:555] Looking for jupyter_config in /usr/local/etc/jupyter,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:555] Looking for jupyter_config in /opt/conda/etc/jupyter,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:555] Looking for jupyter_config in /home/jandal/.jupyter,
[D 2019-09-10 13:01:44.944 SingleUserNotebookApp application:555] Looking for jupyter_config in /home/jandal,
[D 2019-09-10 13:01:44.945 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /etc/jupyter,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:577] Loaded config file: /etc/jupyter/jupyter_notebook_config.py,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /usr/local/etc/jupyter,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /opt/conda/etc/jupyter,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:577] Loaded config file: /opt/conda/etc/jupyter/jupyter_notebook_config.json,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /home/jandal/.jupyter,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:577] Loaded config file: /home/jandal/.jupyter/jupyter_notebook_config.json,
[D 2019-09-10 13:01:44.946 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /home/jandal,
[W 2019-09-10 13:01:44.948 SingleUserNotebookApp configurable:168] Config option `open_browser` not recognized by `SingleUserNotebookApp`.  Did you mean `browser`?,
[D 2019-09-10 13:01:44.953 SingleUserNotebookApp config_manager:98] Paths used for configuration of jupyter_notebook_config: ,
    	/etc/jupyter/jupyter_notebook_config.json,
[D 2019-09-10 13:01:44.953 SingleUserNotebookApp config_manager:98] Paths used for configuration of jupyter_notebook_config: ,
    	/usr/local/etc/jupyter/jupyter_notebook_config.json,
[D 2019-09-10 13:01:44.953 SingleUserNotebookApp config_manager:98] Paths used for configuration of jupyter_notebook_config: ,
    	/opt/conda/etc/jupyter/jupyter_notebook_config.d/jupyterlab.json,
    	/opt/conda/etc/jupyter/jupyter_notebook_config.d/nbdime.json,
    	/opt/conda/etc/jupyter/jupyter_notebook_config.json,
[D 2019-09-10 13:01:44.954 SingleUserNotebookApp config_manager:98] Paths used for configuration of jupyter_notebook_config: ,
    	/home/jandal/.jupyter/jupyter_notebook_config.json,
[I 2019-09-10 13:01:45.136 SingleUserNotebookApp extension:159] JupyterLab extension loaded from /opt/conda/lib/python3.7/site-packages/jupyterlab,
[I 2019-09-10 13:01:45.136 SingleUserNotebookApp extension:160] JupyterLab application directory is /opt/conda/share/jupyter/lab,
[D 2019-09-10 13:01:45.177 SingleUserNotebookApp loader:342] Using default logger,
[D 2019-09-10 13:01:45.177 SingleUserNotebookApp loader:342] Using default logger,
[D 2019-09-10 13:01:45.178 SingleUserNotebookApp loader:342] Using default logger,
[D 2019-09-10 13:01:45.178 SingleUserNotebookApp loader:342] Using default logger,
[D 2019-09-10 13:01:45.178 SingleUserNotebookApp loader:342] Using default logger,
[I 2019-09-10 13:01:45.180 SingleUserNotebookApp singleuser:561] Starting jupyterhub-singleuser server version 1.0.0,
[D 2019-09-10 13:01:45.185 SingleUserNotebookApp _version:60] jupyterhub and jupyterhub-singleuser both on version 1.0.0,
[I 2019-09-10 13:01:45.185 SingleUserNotebookApp notebookapp:1825] Serving notebooks from local directory: /,
[I 2019-09-10 13:01:45.185 SingleUserNotebookApp notebookapp:1825] The Jupyter Notebook is running at:,
[I 2019-09-10 13:01:45.185 SingleUserNotebookApp notebookapp:1825] http://0e9614291632:8888/user/jandal/,
[I 2019-09-10 13:01:45.185 SingleUserNotebookApp notebookapp:1826] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).,
[I 2019-09-10 13:01:45.189 SingleUserNotebookApp singleuser:542] Updating Hub with activity every 300 seconds,
[D 2019-09-10 13:01:45.190 SingleUserNotebookApp singleuser:503] Notifying Hub of activity 2019-09-10T13:01:44.956535Z,
[I 2019-09-10 13:01:45.190 SingleUserNotebookApp log:174] 302 GET /user/jandal/ -> /user/jandal/lab? (@10.0.5.147) 0.69ms,
[I 2019-09-10 13:01:45.233 SingleUserNotebookApp log:174] 302 GET /user/jandal/ -> /user/jandal/lab? (@10.0.5.147) 3.29ms,
[D 2019-09-10 13:01:45.242 SingleUserNotebookApp auth:501] No user identified,
[D 2019-09-10 13:01:45.244 SingleUserNotebookApp auth:832] Redirecting to login url: /hub/api/oauth2/authorize?client_id=jupyterhub-user-jandal&redirect_uri=%2Fuser%2Fjandal%2Foauth_callback&response_type=code&state=eyJ1dWlkIjogIjcyNGFiNTVmMmMyODQ3NTE5NGRhYWRkYzlmNmQyOTY3IiwgIm5leHRfdXJsIjogIi91c2VyL2phbmRhbC9sYWI_In0,
[I 2019-09-10 13:01:45.247 SingleUserNotebookApp log:174] 302 GET /user/jandal/lab? -> /hub/api/oauth2/authorize?client_id=jupyterhub-user-jandal&redirect_uri=%2Fuser%2Fjandal%2Foauth_callback&response_type=code&state=[secret] (@10.0.5.147) 6.83ms,
[D 2019-09-10 13:01:45.299 SingleUserNotebookApp auth:501] No user identified,
[D 2019-09-10 13:01:45.376 SingleUserNotebookApp auth:310] HubAuth cache miss: token:26d114ead3144a6683d988a355911ca8:1GEQWF3rFGBqgaemhlQFLXCCwVxW3D,
[D 2019-09-10 13:01:45.400 SingleUserNotebookApp auth:316] Received request from Hub user {'kind': 'user', 'name': 'jandal', 'admin': True, 'groups': [], 'server': '/user/jandal/', 'pending': None, 'created': '2019-09-05T14:48:26.054495Z', 'last_activity': '2019-09-10T13:01:45.394898Z', 'servers': None},
[I 2019-09-10 13:01:45.400 SingleUserNotebookApp auth:981] Logged-in user {'kind': 'user', 'name': 'jandal', 'admin': True, 'groups': [], 'server': '/user/jandal/', 'pending': None, 'created': '2019-09-05T14:48:26.054495Z', 'last_activity': '2019-09-10T13:01:45.394898Z', 'servers': None},
[D 2019-09-10 13:01:45.400 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:45.402 SingleUserNotebookApp log:174] 302 GET /user/jandal/oauth_callback?code=[secret]&state=[secret] -> /user/jandal/lab? (@10.0.5.147) 104.05ms,
[D 2019-09-10 13:01:45.412 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:45.414 SingleUserNotebookApp handlers:260] Using contents: services/contents,
[I 2019-09-10 13:01:45.429 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab? (jandal@10.0.5.147) 18.49ms,
[D 2019-09-10 13:01:45.478 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:45.480 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:45.481 SingleUserNotebookApp handlers:840] Path main.bb75973800cf065b1994.js served from /opt/conda/share/jupyter/lab/static/main.bb75973800cf065b1994.js,
[D 2019-09-10 13:01:45.485 SingleUserNotebookApp log:174] 304 GET /user/jandal/static/lab/main.bb75973800cf065b1994.js (jandal@10.0.5.147) 8.76ms,
[D 2019-09-10 13:01:45.485 SingleUserNotebookApp handlers:840] Path vendors~main.d490e369048a8684d51e.js served from /opt/conda/share/jupyter/lab/static/vendors~main.d490e369048a8684d51e.js,
[D 2019-09-10 13:01:45.488 SingleUserNotebookApp log:174] 304 GET /user/jandal/static/lab/vendors~main.d490e369048a8684d51e.js (jandal@10.0.5.147) 8.84ms,
[D 2019-09-10 13:01:46.015 SingleUserNotebookApp auth:310] HubAuth cache miss: token:26d114ead3144a6683d988a355911ca8:50390adfbbb14e8db414a28bb955d0e7,
[D 2019-09-10 13:01:46.051 SingleUserNotebookApp auth:316] Received request from Hub user {'kind': 'user', 'name': 'jandal', 'admin': True, 'groups': [], 'server': '/user/jandal/', 'pending': None, 'created': '2019-09-05T14:48:26.054495Z', 'last_activity': '2019-09-10T13:01:46.044347Z', 'servers': None},
[D 2019-09-10 13:01:46.051 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.052 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.054 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/sessions?1568120503944 (jandal@10.0.5.147) 41.18ms,
[D 2019-09-10 13:01:46.056 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.056 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.057 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.058 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.059 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.059 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.060 SingleUserNotebookApp kernelspec:169] Found kernel python3 in /home/jandal/.local/share/jupyter/kernels,
[I 2019-09-10 13:01:46.064 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/kernelspecs?1568120503945 (jandal@10.0.5.147) 8.48ms,
[I 2019-09-10 13:01:46.065 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/terminals?1568120503945 (jandal@10.0.5.147) 8.33ms,
[I 2019-09-10 13:01:46.095 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab/api/settings/?1568120503955 (jandal@10.0.5.147) 36.70ms,
[D 2019-09-10 13:01:46.114 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.114 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.118 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/sessions?1568120504041 (jandal@10.0.5.147) 5.69ms,
[D 2019-09-10 13:01:46.121 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.122 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.124 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.125 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.126 SingleUserNotebookApp kernelspec:169] Found kernel python3 in /home/jandal/.local/share/jupyter/kernels,
[I 2019-09-10 13:01:46.129 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/kernelspecs?1568120504042 (jandal@10.0.5.147) 8.92ms,
[I 2019-09-10 13:01:46.131 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/terminals?1568120504042 (jandal@10.0.5.147) 7.85ms,
[D 2019-09-10 13:01:46.277 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.277 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.292 SingleUserNotebookApp log:174] 200 GET /user/jandal/metrics?1568120504208 (jandal@10.0.5.147) 16.60ms,
[D 2019-09-10 13:01:46.397 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.398 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.418 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.418 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.420 SingleUserNotebookApp handlers:260] Using contents: services/contents,
[D 2019-09-10 13:01:46.456 SingleUserNotebookApp handlers:840] Path base/images/favicon.ico served from /opt/conda/lib/python3.7/site-packages/notebook/static/base/images/favicon.ico,
[D 2019-09-10 13:01:46.456 SingleUserNotebookApp handlers:840] Path components/jquery-ui/themes/smoothness/jquery-ui.min.css served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/jquery-ui/themes/smoothness/jquery-ui.min.css,
[D 2019-09-10 13:01:46.456 SingleUserNotebookApp handlers:840] Path components/jquery-typeahead/dist/jquery.typeahead.min.css served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/jquery-typeahead/dist/jquery.typeahead.min.css,
[D 2019-09-10 13:01:46.456 SingleUserNotebookApp handlers:840] Path style/style.min.css served from /opt/conda/lib/python3.7/site-packages/notebook/static/style/style.min.css,
[D 2019-09-10 13:01:46.457 SingleUserNotebookApp handlers:840] Path components/es6-promise/promise.min.js served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/es6-promise/promise.min.js,
[D 2019-09-10 13:01:46.457 SingleUserNotebookApp handlers:840] Path components/react/react.production.min.js served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/react/react.production.min.js,
[E 2019-09-10 13:01:46.457 SingleUserNotebookApp web:2991] Could not open static file '',
[D 2019-09-10 13:01:46.457 SingleUserNotebookApp handlers:840] Path components/create-react-class/index.js served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/create-react-class/index.js,
[D 2019-09-10 13:01:46.458 SingleUserNotebookApp handlers:840] Path components/requirejs/require.js served from /opt/conda/lib/python3.7/site-packages/notebook/static/components/requirejs/require.js,
[W 2019-09-10 13:01:46.459 SingleUserNotebookApp log:174] 404 POST /user/jandal/git/all_history?1568120504340 (jandal@10.0.5.147) 44.10ms,
[D 2019-09-10 13:01:46.461 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.461 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.462 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.462 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.550 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/nbconvert?1568120504351 (jandal@10.0.5.147) 89.69ms,
[I 2019-09-10 13:01:46.600 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/nbconvert?1568120504376 (jandal@10.0.5.147) 138.65ms,
[D 2019-09-10 13:01:46.602 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.602 SingleUserNotebookApp handlers:840] Path @jupyterlab/theme-light-extension/index.css served from /opt/conda/share/jupyter/lab/themes/@jupyterlab/theme-light-extension/index.css,
[D 2019-09-10 13:01:46.603 SingleUserNotebookApp log:174] 304 GET /user/jandal/lab/api/themes/@jupyterlab/theme-light-extension/index.css (jandal@10.0.5.147) 2.04ms,
[D 2019-09-10 13:01:46.610 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.610 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.672 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.673 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.674 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.674 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.676 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/sessions?1568120504459 (jandal@10.0.5.147) 3.59ms,
[I 2019-09-10 13:01:46.677 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab/api/workspaces/lab?1568120504485 (jandal@10.0.5.147) 4.06ms,
[D 2019-09-10 13:01:46.678 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.678 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.679 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.680 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.691 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/contents/?content=1&1568120504486 (jandal@10.0.5.147) 14.00ms,
[D 2019-09-10 13:01:46.737 SingleUserNotebookApp commands:1994] Fetching URL: https://registry.npmjs.org/@jupyterlab%2Fgit,
[D 2019-09-10 13:01:46.796 SingleUserNotebookApp commands:1619] Node v11.14.0,
    ,
[W 2019-09-10 13:01:46.799 SingleUserNotebookApp commands:2003] Failed to fetch package metadata for '@jupyterlab/git': URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)')),
[D 2019-09-10 13:01:46.801 SingleUserNotebookApp commands:1994] Fetching URL: https://registry.npmjs.org/nbdime-jupyterlab,
[D 2019-09-10 13:01:46.837 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.837 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[D 2019-09-10 13:01:46.839 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.840 SingleUserNotebookApp auth:871] Allowing whitelisted Hub user jandal,
[D 2019-09-10 13:01:46.840 SingleUserNotebookApp auth:744] Setting oauth cookie for 10.0.5.147: jupyterhub-user-jandal, {'path': '/user/jandal/', 'httponly': True},
[I 2019-09-10 13:01:46.848 SingleUserNotebookApp log:174] 200 GET /user/jandal/metrics?1568120504611 (jandal@10.0.5.147) 11.94ms,
[D 2019-09-10 13:01:46.849 SingleUserNotebookApp handlers:840] Path 3.b455b5cc843d6f986fb8.js served from /opt/conda/share/jupyter/lab/static/3.b455b5cc843d6f986fb8.js,
[D 2019-09-10 13:01:46.850 SingleUserNotebookApp log:174] 304 GET /user/jandal/static/lab/3.b455b5cc843d6f986fb8.js (jandal@10.0.5.147) 12.07ms,
[I 2019-09-10 13:01:46.857 SingleUserNotebookApp log:174] 200 GET /user/jandal/api/contents/ml?1568120504625 (jandal@10.0.5.147) 17.50ms,
[I 2019-09-10 13:01:46.857 SingleUserNotebookApp build_handler:43] Build is up to date,
[I 2019-09-10 13:01:46.858 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab/api/build?1568120504328 (jandal@10.0.5.147) 462.62ms,
[W 2019-09-10 13:01:46.864 SingleUserNotebookApp commands:2003] Failed to fetch package metadata for 'nbdime-jupyterlab': URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1056)')),
[I 2019-09-10 13:01:46.870 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab/api/extensions?1568120504471 (jandal@10.0.5.147) 260.07ms,
[I 2019-09-10 13:01:46.871 SingleUserNotebookApp log:174] 200 GET /user/jandal/lab/api/extensions?1568120504499 (jandal@10.0.5.147) 192.67ms,
</pre>
</details>

<details><summary>Browser Output</summary>
Plenty of these:
<pre>
TypeError: "this._iconRegistry is undefined"
    updateItemNode listing.js:1439
    onUpdateRequest listing.js:662
    onUpdateRequest listing.js:659
    processMessage widget.js:492
    x index.js:436
    n index.js:172
    k index.js:486
</pre>
</details>

**Comment by fcollonval** [comment]
Hey @02JanDal, is it possible for you to update Jupyterlab in the docker stack? It looks like, some Jupyterlab packages are incorrect when rebuilding the application with the extensions.

**Comment by telamonian** [comment]
There's a fix now as part of #7192

**Comment by 02JanDal** [comment]
Awesome! Will I need to wait for a new release of JupyterLab (1.2?) or will the changes be pulled in regardless upon running those commands?

Closing this issue meanwhile, will reopen if the problem persists.

**Comment by telamonian** [comment]
@02JanDal The changes will just get pulled in upon the rebuild (this is actually an unfortunate design flaw in the build system that was fixed in Jlab `1.1.x`, but you might as well take advantage of it if you can't just upgrade). You should leave this issue open for the time being. It will get automatically closed once #7192 is merged in.
